src/component/filter/Filter.jsx

```diff
@@ -0,0 +1,208 @@
+                  <input
+                    type="checkbox"
+                    onChange={() => {
+                      filterDispatch({ type: 'WORK_FILTER' });
+                    }}
+                    id="work"
+                    name="work"
+                    value="work"
+                    checked={work}
+                  />
+                  Work
+                </label>
+              </li>
+              <li>
+                <label className="margin-b-5">
+                  <input
+                    type="checkbox"
+                    onChange={() => {
+                      filterDispatch({ type: 'HOMEWORK_FILTER' });
+                    }}
+                    id="homework"
+                    name="homework"
+                    value="homework"
+                    checked={homework}
+                  />
+                  Homework
+                </label>
+              </li>
+              <li>
+                {' '}
+                <label className="margin-b-5">
+                  <input
+                    type="checkbox"
+                    onChange={() => {
+                      filterDispatch({ type: 'CREATIVE_FILTER' });
+                    }}
+                    id="creative"
+                    name="creative"
+                    value="creative"
+                    checked={creative}
+                  />
+                  Creative
+                </label>
+              </li>
+              <li>
+                <label className="margin-b-5">
+                  <input
+                    type="checkbox"
+                    onChange={() => {
+                      filterDispatch({ type: 'EXERCISE_FILTER' });
+                    }}
+                    id="exercise"
+                    name="exercise"
+                    value="exercise"
+                    checked={exercise}
+                  />
+                  Exercise
+                </label>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="filter-elements filter-sort-element">
+          <p>Sort by</p>
+
+          <select
+            className="select-input filter-input-element"
+            onChange={handleSelect}
+          >
+            <option disabled selected value>
+              --select option--
+            </option>
+            <option
+              onChange={() =>
+                filterDispatch({
+                  type: 'DATE_ASC',
+                  payload: 'DATE_ASC',
+                })
+              }
+              selected={sortByState === 'DATE_ASC' ? true : false}
+              // checked={sortByState === 'DATE_ASC' ? true : false}
+              name="sort"
+              value="Date ascending"
+            >
+              Date ascending
+            </option>
+            <option
+              onChange={() =>
+                filterDispatch({
+                  type: 'DATE_DESC',
+                  payload: 'DATE_DESC',
```

Comment:
Since type and payload are the same, could you not use the type for all the calculations. Is payload necessary? Just a doubt